test/spec.yml

```diff
@@ -720,6 +721,26 @@
       out: |-
           "Hello World!"
 
+- test: Test if imagemagick exists in bolt
+  setup:
+      - bn create node8 imagemagick
+      - npm i shelljs
+      - |-
+        cat <<EOF >/home/dockeruser/test/function.js
+        
+        const shell = require('shelljs');
+        exports.handler = async() => {
+          return shell.exec('convert --version');
+        };
+        EOF
+      - bn deploy imagemagick
+  cleanup:
```

Comment:
Setup is different because it's not the core functionality we are testing. I know it seems like a small difference but to me it's a reasonable logical separation.

Reply:
i didn't ask about setup but about cleanup. i expect that section to follow the steps, not the other way around. that's all.